packages/vapor/resources/icons/svg/flag-au.svg

```diff
@@ -0,0 +1 @@
+<svg height="640" viewBox="0 0 10080 5040" width="1280" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><clipPath id="a"><path d="m0 0v1.5h7v1.5h-1zm6 0h-3v3.5h-3v-.5z"/></clipPath><path id="b" d="m0-360 69.421398 215.844981 212.037936-80.30131-125.470868 188.85298 194.985582 115.710885-225.881011 19.650832 31.105109 224.590424-156.198146-164.348792-156.198146 164.348792 31.105109-224.590424-225.881011-19.650832 194.985582-115.710885-125.470868-188.85298 212.037936 80.30131z"/><use id="c" transform="scale(2.1)" xlink:href="#b"/></defs><g transform="scale(840)"><path d="m0 0h12v6h-12z" fill="#00008b"/><path d="m0 0 6 3m0-3-6 3" stroke="#fff" stroke-width=".6"/><path clip-path="url(#a)" d="m0 0 6 3m0-3-6 3" stroke="#f00" stroke-width=".4"/><path d="m3 0v3.5m-3-2h7" stroke="#fff"/><path d="m3 0v3.5m-3-2h7" stroke="#f00" stroke-width=".6"/><path d="m0 3h6v-3h2v4h-8z" fill="#00008b"/></g><g fill="#fff"><use x="2520" xlink:href="#c" y="3780"/><use x="7560" xlink:href="#b" y="4200"/><use x="6300" xlink:href="#b" y="2205"/><use x="7560" xlink:href="#b" y="840"/><use x="8680" xlink:href="#b" y="1869"/><path d="m0-210 54.859957 134.491747 144.861911 10.614684-110.956593 93.735155 34.669628 141.051983-123.434903-76.560236-123.434903 76.560236 34.669628-141.051983-110.956593-93.735155 144.861911-10.614684z" transform="translate(8064 2730)"/></g></svg>
```

Comment:
I think for other svgs we remove the height and width attributes 🤔

Reply:
The EU flag has a width and a height 🤔 , I'll try it out

Reply:
removed attributes in both icons